### PR TITLE
Update postgres example

### PR DESCRIPTION
**Pull request by djKooks**
While testing, I found some missing part in example and updated it.
Also, added on default value in `connect` method, like as in document mentioned `driver : string, default 'psycopg2'`.

Please look on. Thanks.

### CODE REVIEW

File: ibis/sql/postgres/api.py [hunk 1]
```diff
@@ -90,6 +91,7 @@ def connect(
     --------
     >>> import os
     >>> import getpass
+    >>> from ibis.sql.postgres.api import connect
```
Comment (cpcloud) [inline ibis/sql/postgres/api.py]: Can you remove this and change the `connect(...)` line to `ibis.postgres.connect`? That's the correct user-facing API.

**Comment by cpcloud** [comment]
Hm, looks like there's probably a new version of something that doesn't like us calling `engine.execute('VACUUM ANALYZE')`, though I'm curious why it's only failing on the Python 3.6 build.

File: ibis/sql/postgres/api.py (hunk 1 above)
Comment (djKooks) [reply ibis/sql/postgres/api.py]: @cpcloud Okay. I'll follow up in next monday. Thanks.

Comment (djKooks) [reply ibis/sql/postgres/api.py]: @cpcloud Sorry for late. I updated it. I'll look on build fail issue, too.

**Comment by cpcloud** [comment]
@djKooks can you rebase on top of master and then push? you need to pick up the latest changes to pass the CI testing.

**Comment by djKooks** [comment]
@cpcloud Well, it seems it causes more failures here. Is other people works well with this change?

**Comment by cpcloud** [comment]
@djKooks This change is invalid, because we use the environment variables that you've changed here in our tests.

**Pull request closed without merging**